chapters/dashboard/flow.md

```diff
@@ -0,0 +1,33 @@
+## Flow Design GUI
+
+##### Our GUI is a visual interface that makes composing flows as easy as browsing Facebook. Design, configure and deploy your flows right from the browser!
+
+### Features
+
+-  Build your own custom Flows per Drag & Drop on the canvas leveraging templates, YAML files or by starting from scratch with empty pods and flows.
+-  Deploy your flow with one click directly through JinaD
+-  Export your flow as YAML or PNG 
+
+
+### Before you start
+
+Before you start, you have to [launch JinaD and connect it to the Dashboard](connect-jinaD.md)
+
+### Composing a flow
+
+![compose flow](flowCompose.gif)
```

Comment:
I think its really good that we have gifs but we might still want some textual description to accompany it

Reply:
Added descriptions